pkg/monitoring/configmaps.go

```diff
@@ -24,12 +25,19 @@ func LoggingConfigMaps(managedClusterName string) []*corev1.ConfigMap {
 	if err != nil {
 		zap.S().Debugf("New logging config map %s is giving error %s", filebeatindexconfig.Name, err)
 	}
-	filebeatconfig, err := createLoggingConfigMap(constants.LoggingNamespace, "filebeat-config", "filebeat.yml", FilebeatConfigData, filebeatLabels)
+
+	filebeatConfigData := FilebeatConfigDataDocker
```

Comment:
Do we need to worry about the "unknown" case?

Reply:
Not currently. Currently we default to docker and explicitly check for containerd. So, in the case of a failure to obtain the container runtime, it will default to docker in call cases. Ideally, this container runtime string would be encapsulated in a struct but to keep things simple, especially since this is all moving to the verrazzano repo, I just parse the string where I need it.